test/StronglyTypedId.Tests/StronglyTypedId.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
This should work if we're building for .NET Core 3.1 shouldn't it?

Reply:
Yes, it should be fine

Reply:
I mean, we want our tests to run on .netcore 3, but it looks like that broke the build? Ideally, I'd like the tests to target multiple platforms to confirm it all works correctly 🙂

Reply:
Oh, yes, AppVeyor seems to be configured to build with .NET Core 2.2. On my local machine it was obviously building correctly with .NET Core 3.1

Reply:
Looks like appveyor is missing the `image` tag for some reason! 

If you add it in (as in here):
https://github.com/andrewlock/PwnedPasswords/blob/2e3f117bcd3939cd53e25ab4e1ca15cea918398e/appveyor.yml#L2

Then we should be able to update the tests to multi-target all supported runtimes to be safe 🙂

```xml
<TargetFrameworks>netcoreapp2.1,netcoreapp3.0,netcoreapp3.1</TargetFrameworks>
```

Thanks!

Reply:
Done, build succeeded

Reply:
Perfect, thanks!